### PR TITLE
Fix background color doesn't match below api 23 version

### PR DESCRIPTION
**Pull request by ehdrms2034**
https://github.com/sangcomz/StickyTimeLine/issues/19#issuecomment-544216732
i saw preludezdev's opinion, so i resolved it

* added directory named 'values-21','values-23'
* modified proper default background color in color.xml respectively

### CODE REVIEW

File: stickytimelineview/src/main/res/values-v21/colors.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="colorDefaultBackground">#EEEEEE</color>
+    <color name="colorDefaultTitle">#414fca</color>
+    <color name="colorDefaultSubTitle">#d16767</color>
+    <color name="colorDefaultLine">#51ae45</color>
+    <color name="colorDefaultCircle">#51ae45</color>
+    <color name="colorDefaultStroke">#F9F9F9</color>
```
Comment (sangcomz) [inline stickytimelineview/src/main/res/values-v21/colors.xml]: These values ​​are the same as other versions, so you don't need to declare them again.

File: stickytimelineview/src/main/res/values-v23/colors.xml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="colorDefaultBackground">#F9F9F9</color>
+    <color name="colorDefaultTitle">#414fca</color>
+    <color name="colorDefaultSubTitle">#d16767</color>
+    <color name="colorDefaultLine">#51ae45</color>
+    <color name="colorDefaultCircle">#51ae45</color>
+    <color name="colorDefaultStroke">#F9F9F9</color>
```
Comment (sangcomz) [inline stickytimelineview/src/main/res/values-v23/colors.xml]: These values ​​are the same as other versions, so you don't need to declare them again.

File: stickytimelineview/src/main/res/values-v21/colors.xml (hunk 1 above)
Comment (ehdrms2034) [reply stickytimelineview/src/main/res/values-v21/colors.xml]: Oh thx, i didn't know that values can overlap
i removed the code you pointed

**Comment by ehdrms2034** [comment]
Due to your feedback, i could make code better, thx
could u check this code again?

**Pull request closed without merging**